app/src/App.tsx

```diff
@@ -1,142 +1,31 @@
-			.then((accessToken: string) => {
-				if (accessToken) {
-					return app.reAuthenticate().then(() => setIsAuthenticated(true));
-				}
-			})
-			.then(() => setIsLoading(false));
-	}, []);
-
-	const onMobile = isMobile();
-
-	const textStyle = {
-		fontFamily: 'Roboto, Arial, Helvetica, sans-serif',
-		fontSize: 22,
-		fontWeight: 100,
-	};
-
-	return (
-		<AppProvider i18n={translations}>
-			<div
-				style={{
-					display: 'flex',
-					alignItems: 'center',
-					justifyContent: 'center',
-					height: '100%',
-					width: '100%',
-					overflow: 'hidden',
-					position: 'absolute',
-				}}
-			>
-				<Snackbar
-					anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
-					open={snackBarOpen}
-					autoHideDuration={6000}
-					onClose={handleCloseSnackBar}
-					message={snackBarMessage}
-				/>
-				<Paper
-					elevation={onMobile ? 0 : 3}
-					style={{
-						padding: onMobile ? 10 : 20,
-						position: 'relative',
-						minHeight: 500,
-						...(onMobile
-							? { height: '100%', width: '100%', overflow: 'scroll' }
-							: { width: 500 }),
-					}}
-				>
-					<div
-						style={{
-							fontFamily: 'Roboto, Arial, Helvetica, sans-serif',
-							fontSize: 28,
-							fontWeight: 700,
-							textAlign: 'center',
-							marginTop: onMobile ? 10 : 20,
-							marginBottom: onMobile ? 10 : 40,
-						}}
-					>
-						Project Name
-					</div>
-					{isLoading ? (
-						<div
-							style={{
-								position: 'fixed',
-								right: 'calc(50vw - 22px)',
-								top: 'calc(50vh - 22px)',
-							}}
-						>
-							<CircularProgress />
-						</div>
-					) : isAuthenticated ? (
-						<div
-							style={{ ...textStyle, margin: '60px auto', textAlign: 'center' }}
-						>
-							Congrats, you're now logged in!
-							<Button onClick={() => setIsAuthenticated(false)}>Logout</Button>
-						</div>
-					) : (
-						<div>
-							<div style={{ ...textStyle, fontSize: 16, padding: '0 20px' }}>
-								Already have an account?
-							</div>
-							<Login authenticate={authenticate} />
-							<div
-								style={{
-									...textStyle,
-									margin: '30px auto',
-									textAlign: 'center',
-								}}
-							>
-								OR
-							</div>
-							<div style={{ ...textStyle, fontSize: 16, padding: '0 20px' }}>
-								Register as a new user
-							</div>
-							<Registration authenticate={authenticate} />
-						</div>
-					)}
-				</Paper>
-			</div>
-		</AppProvider>
-	);
+import React from 'react'
+import { Navigate, Route, Routes } from 'react-router-dom'
+
+import AuthContext from 'STARTER/contexts/auth'
+
+import Home from 'STARTER/pages/home'
+import Dashboard from 'STARTER/pages/dashboard'
+import Protected from 'STARTER/components/protected'
+
+
+const App = () => {
+
+  return (
+    <AuthContext>
+      <Routes>
+        <Route path='/home' element={<Home/>}/>
+        <Route
+          path='/dashboard'
+          element={
+            <Protected>
+              <Dashboard/>
```

Comment:
Is Dashboard needed in the starter? The login page seems to work differently than other apps and we don't automatically get rerouted to the dashboard on login.

Reply:
I added the dashboard as an example of protected routing, and can further update it so it's redirected after login!

Reply:
I pushed a commit that updates it and I think makes the routing slightly more logical!